user-guide/Reference/Soft-launch_options/Overview_of_Soft_Launch_Options.md

```diff
@@ -363,3 +345,4 @@ Uses an embedded web browser for Automation script execution instead of the nati
 Enables the Cube UI for [User-Defined APIs](xref:UD_APIs).
 
 - **Minimum version**: 10.3.5
+- **Release version**: 10.3.6
```

Comment:
As long as the release note to add this hasn't actually been merged yet, I'd prefer not to mention a release version as absolutely certain:

```suggestion
- **Estimated release version**: 10.3.6
```